dashboard/app/views/reports/csp_pd_responses.csv.erb

```diff
@@ -0,0 +1,10 @@
+<%=
+CSV.generate do |csv|
```

Comment:
Do you know what Content-Disposition header this is going to be served with?

Put another way, is the browser going to try to download or display this content?